packages/slate-editor/src/components/SearchInput/SearchInput.tsx

```diff
@@ -98,11 +96,11 @@ export function SearchInput<T = unknown>({
     useRootClose(rootRef, () => setOpen(false));
 
     useEffect(() => {
-        const unlisten = EventsEditor.addEventListener(editor, 'contact-dialog-searched', () =>
-            search(query),
-        );
```

Comment:
I had to refactor this, as the implementation felt wrong. 

I needed this functionality for story embeds and story bookmarks. And this call here was introduced in the generic `SearchInput` (used by all placeholders), bound to a contact-specific event name.

Reply:
Yeah, it seems weird looking back at it, good work 👍